### DISCUSSION TITLE
【0x01 Project】基于 eth.build 可视化理解区块链基础原理之一

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#3] [Projects]
地址：https://eth.build

### 资料

https://mp.weixin.qq.com/s/wAoaFu6ql0HfR-LETGQ2DA

https://www.youtube.com/playlist?list=PLJz1HruEnenCXH7KW7wBCEBnBLOVkiqIi

![image](https://user-images.githubusercontent.com/12784118/140302325-6f3809d5-5825-4fb1-8425-e8cda310550e.png)

先看1、2、3、4！

### 辅助学习视频

[微芒专题 | 14天区块链极速入门](https://ke.qq.com/course/3614841)

（可免费学习，在群里咨询「一点知道」辅助学习课程获取方式！）

### 任务

用eth.build实现和以太坊交互 —— 如何发送 eth？

难度：☆☆（十星制）
---
完成的同学请在下面👇回帖接龙！

**Comment by lysion** [comment]
https://eth.build/build#189f7d9f73a7eae10f4c9d0c353628dcf1bdb0ee70d8f3cd838ea27fa03155a1

**Comment by sunliying2020** [comment]
https://eth.build/build#5edb9018552481cb71f62cae43dcbf17e13145d443bafc071e8481571da33027

**Reply by sunliying2020** [reply]
https://shimo.im/docs/TCcwcygwKhWTtkdJ/ 《eth.build学习笔记》

**Reply by leeduckgo** [reply]
👍👍👍

**Comment by hechty** [comment]
https://eth.build/build#fcd1557c817cac917e1a4a2fdd35215ec79def88a3814797626c15db18bd39b0

学习笔记如下：
https://mray4uxds2.feishu.cn/docs/doccnRTsGa53GvPDA3ufCbT16ih

**Comment by Dream4ever** [comment]
https://eth.build/build#f0c7f493d0b3ac62a578701c224aa4a236a7cb6916fe07b981749db0becd990c

**Comment by gofenix** [comment]
https://eth.build/build#cbafe2a2404fb6ebdd2a3875a685583ba416a5d306576d16e037d1da6141dec1

**Comment by xFANx** [comment]
https://eth.build/build#4726dbe7e1370aef3819eb2ace1ee61f93e0500a533871a5c5c59bc76c958641
